src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+document.querySelectorAll('li').forEach((li) => {
+  if (li.querySelector('ul')) {
+    const span = document.createElement('span');
+
+    span.append(li.firstChild);
+    li.prepend(span);
+  }
+});
+
+document.querySelector('.tree').addEventListener('click', (e) => {
+  if (e.target.tagName === 'SPAN') {
+    const subList = e.target.nextElementSibling;
+
+    if (subList) {
+      subList.style.display =
+        subList.style.display === 'none' ? 'block' : 'none';
```

Comment:
There seems to be an issue with the assignment of 'subList.style.display'. The assignment is split across two lines, which might cause a syntax error. Consider combining these lines into a single line for proper assignment.